pkgs/development/tools/click/default.nix

```
@@ -0,0 +1,92 @@
+{ lib
+, fetchFromGitLab
+, buildPythonApplication
+, autoreconfHook
+, debian
+, automake
+, perl
+, libtool
+, vala
+, pkg-config
+, libgee
+, json-glib
+, properties-cpp
+, gobject-introspection
+, getopt
+, setuptools
+, pygobject3
+, wrapGAppsHook
+}:
+
+buildPythonApplication {
+  pname = "click";
+  version = "22-02-2023";
+
+  src = fetchFromGitLab {
+    owner = "ubports";
+    repo = "development/core/click";
+    rev = "aaf2735e8e6cbeaf2e429c70136733513a81718a";
+    sha256 = "sha256-pNu995/w3tbz15QQVdVYBnWnAoZmqWj1DN/5PZZ0iZw=";
+  };
+
+  configureFlags = [
+    "--with-systemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
+    "--with-systemduserunitdir=${placeholder "out"}/usr/lib/systemd/user"
+  ];
+
+  preConfigure = ''
+    export click_cv_perl_vendorlib=$out/${perl.libPrefix}
+    export PYTHON_INSTALL_FLAGS="--prefix=$out"
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    cp -r {bin,click_package,build,lib} $out
+    # cp -r . $out
+  '';
+
+  preFixup = ''
+    makeWrapperArgs+=(--prefix GI_TYPELIB_PATH : "${placeholder "out"}/lib/click" --prefix LD_LIBRARY_PATH : "${placeholder "out"}/lib")
+  '';
+
+  top_builddir = "$(pwd)";
+
+  click_cv_perl_vendorlib = "${placeholder "out"}/lib";
+
+  format = "other";
+
+  nativeBuildInputs = [
+    autoreconfHook
+    automake
+    perl
+    libtool
+    pkg-config
+    gobject-introspection.dev
```

Explicit `.dev` again.

```suggestion
    gobject-introspection
```